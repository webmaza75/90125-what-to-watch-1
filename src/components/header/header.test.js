import React from 'react';
import renderer from 'react-test-renderer';
import {StaticRouter} from 'react-router';

import Header from './header.jsx';
// import user from '../../mocks/user.js';

it(`Header correctly renders`, () => {
  const context = {};
  const header = renderer
    .create(<StaticRouter location="someLocation" context={context}>
      <Header
        user={null}
      />
    </StaticRouter>)
    .toJSON();

  expect(header).toMatchSnapshot();
});
