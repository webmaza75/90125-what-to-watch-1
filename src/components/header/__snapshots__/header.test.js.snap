// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header correctly renders 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link "
      href="main.html"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <a
      className="user-block__link"
      href="/login"
      onClick={[Function]}
    >
      Sign in
    </a>
  </div>
</header>
`;
